dbt/adapters/glue/connections.py

```diff
@@ -101,7 +101,7 @@ def get_response(cls, cursor) -> AdapterResponse:
     def get_result_from_cursor(cls, cursor: GlueCursor, limit: Optional[int]) -> agate.Table:
         data: List[Any] = []
         column_names: List[str] = []
-        if cursor.description is not None:
+        if not cursor.description:
```

Comment:
This fixes the following error when running the test

```
E           dbt.exceptions.DbtRuntimeError: Runtime Error
E             Compilation Error in model model (models/model.sql)
E               'NoneType' object is not iterable
E               
E               > in macro run_query (macros/etc/statement.sql)
E               > called by model model (models/model.sql)
```

However, even if it's fixed, the test fails by running getTable API to the view, which is not registered to Glue Catalog database.